client/src/js/services/PatientInvoiceService.js

```diff
@@ -120,6 +120,7 @@ function PatientInvoiceService(Modal, util, Session, Api) {
       { field: 'service_id', displayName: 'FORM.LABELS.SERVICE' },
       { field: 'user_id', displayName: 'FORM.LABELS.USER' },
       { field: 'reference', displayName: 'FORM.LABELS.REFERENCE' },
+      { field: 'debtor_uuid', displayName: 'FORM.LABELS.CLIENT' },
```

Comment:
The debtor_uuid property was to add to formatFilterParameters to allow the FORM.LABELS.CLIENT display to the case the filtering is performed relative to the Debitor